java/java-jwt/security/jwt-hardcode.java

```diff
@@ -48,6 +48,7 @@ public static void main( String[] args )
 
     public void bad2() {
         try {
+            // ruleid: java-jwt-hardcoded-secret
```

Comment:
Yeah wait a sec, should we really be flagging this? This is not guaranteed to be a constant.

Reply:
How would it not be, if the value of `secret` is never touched again? I may be unclear on Java.

Reply:
The problem is that we _do not know_ if `secret` will ever be touched again, given its visibility another object could surely modify it. And const-prop is supposed to be a _must_ analysis, so if we report that something is constant when it may not be, that is a bug.